Documentation/gettingstarted/host-services.rst

```diff
@@ -1,73 +0,0 @@
-.. only:: not (epub or latex or html)
```

Comment:
Should this be completely removed? It may make sense to replace this page with a link to the updated "kpr" docs for a version or two, allowing ppl to understand where the page went.

Reply:
The page didn't have any unique content. Everything was already in the KPR section, so decided just to remove it.